content/docs/04-clusters/1-new-clusters/03-azure.md

```diff
@@ -84,15 +88,22 @@ The following steps need to be performed to provision a new Azure cluster:
     | **Region** | Select a region in Azure in which the cluster should be deployed.|
     | **Resource Group** | Select the resource group in which the cluster should be deployed.|
     | **SSH Key** | Public key to configure remote SSH access to the nodes.|
-    | **Static Placement** | By default, Palette uses dynamic placement, wherein a new VPC with a public and private subnet is created to place cluster resources for every cluster.<br /> These resources are fully managed by Palette and deleted when the corresponding cluster is deleted. Turn on the **Static Placement** option if it's desired to place resources into preexisting VPCs and subnets. If the user is making the selection of **Static Placement** of resources, the following placement information needs to be provided:
-    || **Virtual Network**: Select the virtual network from dropdown menu.
-    || **Control plane Subnet**: Select the control plane network from the dropdown menu.
-    || **Worker Network**: Select the worker network from the dropdown.
-
-5. Make the choice of updating the worker pool in parallel.
-
+    | **Static Placement** | By default, Palette uses dynamic placement, wherein a new VPC with a public and private subnet is created to place cluster resources for every cluster. These resources are fully managed by Palette and deleted when the corresponding cluster is deleted. <br /> Turn on the **Static Placement** option if it is desired to place resources into preexisting VPCs and subnets. If the user is making the selection of **Static Placement** of resources, the following placement information needs to be provided:
+    ||**Virtual Resource Group** : The logical container for grouping related Azure resources.
+    ||**Virtual Network**: Representation of your own network in the cloud.
+    ||**CIDR Block**: To assign Internet Protocol (IP) addresses that improve address distribution efficiency.
+    ||**Control plane Subnet**: For Azure resource management <br /> Name <br /> CIDR Block <br /> Security Group Name
+    ||**Worker Network**: <br /> Name <br /> CIDR Block <br /> Security Group Name
+    |**Update worker pools in parallel**| Check the box to concurrently update the worker pools.|
+
+<InfoBox>
+
+If the Palette [cloud account](/clusters/new-clusters/azure#creatinganazurecloudaccount) is created with **Disable Properties** and 
+**Static Placement** checked the network informations from user's Azure account will not be imported to palette account. Hence all the network informations given in the above table needs to be filled manually by the user (no drop down menu will be available).
+
+</InfoBox>

```

Comment:
I'd enhance this note with the fact that you'd have to manually input the information for **Control plane Subnet** and **Worker Network**
```
Name <br /> CIDR Block <br /> Security Group Name
```
if these conditions are met

Reply:
done